modules/ai/assets/js/editor/utils/editor-integration.js

```diff
@@ -65,11 +65,19 @@ export const renderLayoutApp = ( options = {
 	const rootElement = document.createElement( 'div' );
 	document.body.append( rootElement );
 
+	const bodyStyle = window.elementorFrontend.elements.$window[ 0 ].getComputedStyle( window.elementorFrontend.elements.$body[ 0 ] );
+
 	ReactDOM.render(
 		<LayoutApp
 			mode={ options.mode }
 			isRTL={ isRTL }
 			colorScheme={ colorScheme }
+			currentContext={ {
+				body: {
+					backgroundColor: bodyStyle.backgroundColor,
+					backgroundImage: bodyStyle.backgroundImage,
```

Comment:
what about a background image that doesn't have a public URL?

Reply:
It's not a problem, the context if for better color contrast, so, for a local image it will not run this improvement.